pkg/issuer/vault/issue.go

```diff
@@ -131,7 +131,12 @@ func (v *Vault) requestTokenWithAppRoleRef(client *vault.Client, appRole *v1alph
 		"secret_id": secretId,
 	}
 
-	url := "/v1/auth/approle/login"
+	authPath := appRole.Path
+	if authPath == "" {
+		authPath = "approle"
```

Comment:
👍 this is okay for now. We will look at moving defaulting etc. out after the webhooks PR has merged